foundations/introduction/how_this_course_will_work.md

```diff
@@ -1,45 +1,44 @@
 ### Welcome to The Odin Project
 
-The Odin Project is an open source community dedicated to providing the best information sources to take you from zero to full-stack developer.  More information can be found at The Odin Project's [about page](https://www.theodinproject.com/about).
+The Odin Project is an open-source community dedicated to providing the best information sources to take you from zero to a full-stack developer.  More information can be found at The Odin Projects [about page](https://www.theodinproject.com/about).
 
 ### Introduction
 
-In this unit, we'll learn about how the web works and start thinking about the basics of computer and web programming.
+In this unit, we’ll learn about how the web works and start thinking about the basics of computer and web programming.
```

Comment:
Is there a reason you're using a "smart quote" instead of a standard single quote?